content/en/docs/concepts/architecture/nodes.md

```diff
@@ -81,25 +81,26 @@ The node condition is represented as a JSON object. For example, the following r
 
 If the Status of the Ready condition remains `Unknown` or `False` for longer than the `pod-eviction-timeout`, an argument is passed to the {{< glossary_tooltip text="kube-controller-manager" term_id="kube-controller-manager" >}} and all the Pods on the node are scheduled for deletion by the Node Controller. The default eviction timeout duration is **five minutes**. In some cases when the node is unreachable, the apiserver is unable to communicate with the kubelet on the node. The decision to delete the pods cannot be communicated to the kubelet until communication with the apiserver is re-established. In the meantime, the pods that are scheduled for deletion may continue to run on the partitioned node.
 
-In versions of Kubernetes prior to 1.5, the node controller would [force delete](/docs/concepts/workloads/pods/pod/#force-deletion-of-pods)
-these unreachable pods from the apiserver. However, in 1.5 and higher, the node controller does not force delete pods until it is
-confirmed that they have stopped running in the cluster. You can see the pods that might be running on an unreachable node as being in
-the `Terminating` or `Unknown` state. In cases where Kubernetes cannot deduce from the underlying infrastructure if a node has
-permanently left a cluster, the cluster administrator may need to delete the node object by hand.  Deleting the node object from
-Kubernetes causes all the Pod objects running on the node to be deleted from the apiserver, and frees up their names.
-
-In version 1.12, `TaintNodesByCondition` feature is promoted to beta, so node lifecycle controller automatically creates
+The node controller does not force Pod deletion until it has confirmed that those Pods
+have stopped running in your cluster. You can see the pods that might be running on
+an unreachable node as being in the `Terminating` or `Unknown` state. In cases where
+Kubernetes cannot deduce from the underlying infrastructure if a node has permanently
+left a cluster, the cluster administrator may need to delete the node object by hand.
+Deleting the node object from Kubernetes causes all the Pod objects running on the
+node to be deleted from the apiserver, and frees up their names.
+
+If you have the `TaintNodesByCondition`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) enabled
+(the default), then the node lifecycle controller automatically creates
```

Comment:
nit: If the TaintNodesByCondition is enabled by default, do you need lines 92- beginning of 94? 
Same comment could be applied to lines  231-233 and lines 285-286.

Reply:
That  sounds like a good idea for an improvement, maybe for a future PR.

Reply:
When features are beta, I think it helps to let readers know how to opt out of them.